tags.go

```diff
@@ -30,3 +30,22 @@ func GetTagValue(tags []string, key string, separator string) string {
 
 	return ""
 }
+
+// Add adds a new tag at the beginning of the comma-separated tag list
+func (tags Tags) Add(newTag string) Tags {
+	return Tags(strings.Join(append(tags.Split(), newTag), ", "))
+}
+
+// RemoveFromListByKey removes the first item found from the tag list that contains the specified key
+func (tags Tags) RemoveFromListByKey(key string) Tags {
```

Comment:
What you think about `RemoveByKey`?

Reply:
Yep that's what I suggested to Ian too! I can update it 😄